src/GSMClient.cpp

```diff
@@ -446,3 +432,19 @@ void GSMClient::handleUrc(const String& urc)
     }
   }
 }
+
+int GSMClient::state() {
+	return _state;
```

Comment:
Please use the indentation style established by the existing code. In all your modifications to this file, you used true tabs for indentation, while the existing code uses two space indentation.

Reply:
@johncpang thanks for resolving my other suggestions about the brace style. However, the issue of the use of true tabs for indentation instead of spaces remains unresolved.